apps/builder/app/builder/features/ai/ai-command-bar.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-internal-modules */
```

Comment:
Better add exceptikn jn plugin

Reply:
Have no idea why we add it at all. Will leave as is. We can cleanup config later